src/renderers/testing/ReactTestMount.js

```diff
@@ -149,24 +149,22 @@ var ReactTestMount = {
 
   render: function(
     nextElement: ReactElement<any>,
-    mockConfig: TestRendererMockConfig,
+    mockConfig?: TestRendererMockConfig,
```

Comment:
I think we put `?` before the type elsewhere, like ?Stuff. Not sure if there's two ways of doing the same thing?